content/blog/2025-02-03-submit_posit_conf/index.en.md

```diff
@@ -0,0 +1,72 @@
+
+💡 **Nic Crane**: Check out the [Posit blog post which tells you what they're looking for](https://posit.co/blog/how-we-build-the-positconf-program/) and has examples of past submissions.  Before recording, I try to get myself pumped up with energy by doing a dummy run pretending my idea is the most interesting thing I've heard in my entire life, with exaggerated enthusiasm, so then the normal version I do immediately afterwards has a bit of bounce to it.
+
+📽  *[Placeholder for a tip from another R-Ladies member]*
+
+🎤  *[Placeholder for a tip from another R-Ladies member]*
+
+💡  *[Placeholder for a tip from another R-Ladies member]*
+
+
+## **What Tech Should You Use?**
+
+*[Placeholder for Saya's content]*
+
+## **How to Structure Your Video**
+
+Here's a simple way to approach your **one-minute** video:
+
+### **1. Start with the Basics**
+
+Introduce yourself and your idea. Pull from the abstract of your talk and expand slightly to explain what your talk is about.
+
+### **2. Share Why It Matters**
+
+Why is your idea important to you? Why does it excite you? People love to see genuine enthusiasm—it's contagious! Highlight why your topic could be interesting or helpful to the audience.
+
+### **3. Keep It Conversational**
+
+Talk as if you're **explaining to a friend,** not reading from a script. Keep it engaging and natural.
+
+### **4. Speak Clearly & With Energy**
+
+Passion is contagious! **Slow down a bit,** use your voice expressively, and **have fun with it.**
+
+### **5. Don't Worry About Perfection**
+
+Your video doesn't need to be flawless—just **authentic**. If selected, you'll get speaker training!
+
+### **6. Humor and Creativity are Welcome**
+
+If your idea lends itself to something quirky or funny, don't shy away. Even if your topic seems niche, an engaging presentation can make it appealing to everyone.  But don't worry if you're not naturally funny either - this is only optional.
+
+## **Tips for Success**
+
+* **Pep Talk Yourself:** Before recording, remind yourself that you're great at what you do. Boosting your confidence will help you project positivity and ease.  
+* **Keep It Simple:** Your video doesn't need to be polished or perfect. Focus on your enthusiasm and message.  
+* **Check Out Examples:** Watch [other video abstracts](https://posit.co/blog/how-we-build-the-positconf-program/) for inspiration.
+
+## **Need Feedback?**
+
+If you'd like input on your abstract or video, we're happy to help! Share a link to your draft (Google Drive or similar and if you're submitting a video then also include a link to a Google Doc where a reviewer can put their comments), and a reviewer will take a look.
```

Comment:
```suggestion
If you would like input on your abstract or video, we are happy to help! Share a link to your draft (Google Drive or similar and if you're submitting a video then also include a link to a Google Doc where a reviewer can put their comments), and a reviewer will take a look.
```

yes, let's discuss this more. is the process for video review the same as abstract review? does it go to the same reviewer? either way, direct the user more specifically to where they should share or submit.

Reply:
also one more thought here. what is the expectation to get feedback on your video submission? is it that you record it, submit it for review, and then record again? or that you live workshop talking points? or that you have talking points written down that you discuss?

Reply:
I think to ensure we're creating minimal commitment for reviewers, we should set the expectation record -> get review -> re-record, but now we're asking for permission to share email addresses, if reviewers want to offer extra feedback then they can choose to do so.

Reply:
yeah that sounds fine to me. i would be explicit that the "draft" is an initial recording that they get feedback on.